code/ship/ship.cpp

```diff
@@ -18990,36 +18990,36 @@ ship_subsys* ship_get_subsys_for_submodel(ship* shipp, int submodel)
 /**
  * Is the requested type of ship arriving in the requested stage?
  *
- * Dock leaders & followers need to be handled separately such that
- * leaders handle the warp process, but followers need to behave as though
- * warping with respect to targeting, drawing thrusters, radar, AI
- * targeting, etc
+ * Dock leaders (plus single ships) & dock followers need to be handled
+ * separately such that leaders+singles handle the warp process, but
+ * followers need to behave as though they are warping with respect to
+ * HUD targeting, drawing thrusters, radar, AI targeting, etc
  *
  * @param stage Check one of stage1, stage2 or both
- * @param dock_leader_only only the dock leader, or leader/followers/non-docked?
+ * @param dock_leader_or_single_only check dock-leaders+singles, or dock-followers?
  *
  * @return is the ship arriving, bool
  */
-bool ship::is_arriving(ship::warpstage stage, bool dock_leader_only)
+bool ship::is_arriving(ship::warpstage stage, bool dock_leader_or_single_only)
```

Comment:
Can you remove the word "only" from the parameter?

Good commit, and good change in the comments, but I'm surprised that the only changes necessary were within the function itself.  Do all the function calls use the boolean correctly?

Reply:
sure, I'll remove `_only`

Yes, all the function calls use the boolean correctly, I've rechecked them and rerun my test mission to confirm.

Reply:
Excellent.   Then I think we can finally call this done!